website/docs/registry/providers/tiers.html.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: "registry"
+page_title: "Terraform Registry - Provider Tiers
+sidebar_current: "docs-registry-provider-tiers
+description: |-
+  Published Provider tiers in the Terraform Registry
+---
+
+# Provider Tiers
+
+There are three tiers of providers in the Terraform Registry:
+
+* **Official Providers** - are built, signed, and supported by HashiCorp. Official Providers can typically used without providing
+  provider source information in your Terraform configuration.
```

Comment:
I believe this is going to be a requirement or warning in 0.13, and that users should use the upgrade tool to specify the explicit source, @hashicorp/terraform-core can confirm though.

Reply:
Not for official providers - if it's in the `hashicorp` namespace, source is not required.